src/hooks/useComplianceState.tsx

```diff
@@ -0,0 +1,86 @@
+import { shallowEqual, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { ComplianceStatus } from '@/constants/abacus';
+import { CLOSE_ONLY_GRACE_PERIOD, ComplianceStates } from '@/constants/compliance';
+import { STRING_KEYS } from '@/constants/localization';
+import { isMainnet } from '@/constants/networks';
+
+import { LinkOutIcon } from '@/icons';
+
+import { getComplianceStatus, getComplianceUpdatedAt, getGeo } from '@/state/accountSelectors';
+import { getSelectedLocale } from '@/state/localizationSelectors';
+
+import { isBlockedGeo } from '@/lib/compliance';
+
+import { useURLConfigs } from '.';
+import { useStringGetter } from './useStringGetter';
+
+export const useComplianceState = () => {
+  const stringGetter = useStringGetter();
+  const complianceStatus = useSelector(getComplianceStatus, shallowEqual);
+  const complianceUpdatedAt = useSelector(getComplianceUpdatedAt, shallowEqual);
+  const geo = useSelector(getGeo, shallowEqual);
```

Comment:
qq: is `geo` just a string? shallowEqual might not be necessary

Reply:
I think the same for `complianceUpdatedAt` and possible `complianceStatus` (unless this is a kotlin enum)

Reply:
good point! complianceStatus is enum, but geo, and complianceUpdatedAt probably don't need shallowEqual